Vic2ToHoI4/Data_Files/Configurables/CountryNamesMappings.txt

```diff
@@ -0,0 +1,14 @@
+# These mappings define how country names are assigned.
+#
+# For the specified HoI4 ideology, the name comes from the matching Vic2 government.
+# A Vic2 tag can be required as well: name_mapping = { vic2_gov = democracy vic2_tag = TAG hoi4_ideology = democratic }
+# This will ensure the rule is applied only to the matching country, allowing for customizations.
+#
+# The rules are scanned in order, and the first matching rule is the one that is applied.
```

Comment:
You could also add a note that multiple entries of the same type in a mapping are not supported.